### PR TITLE
added msteams meetings v2.0.0

### PR DESCRIPTION
**Pull request by levb**
Added MSTeams Meetings v2.0.0 to the marketplace

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -4475,6 +4475,93 @@
...
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-msteams-meetings-v2.0.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-msteams-meetings/releases/tag/v2.0.0",
+    "hosting": "",
+    "author_type": "mattermost",
+    "release_stage": "production",
+    "enterprise": false,
```
Comment (agarciamontoro) [inline plugins.json:4486]: Should this be true?

Comment (wiersgallak) [reply plugins.json:4486]: MS team meetings is available in our Professional and Enterprise packages (not Starter). Same for Cloud & Self-hosted

**Review by agarciamontoro — APPROVED** [review]
Thanks, Lev!

File: plugins.json (hunk 1 above)
Comment (wiersgallak) [reply plugins.json:4486]: THanks!

**Review by wiersgallak — APPROVED** [review]

**Review by catalintomai — APPROVED** [review]

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Tested upgrade path (including for older server version)
- Tested install path
- No change to icon
- Version number is correct
- Configured and enable the plugin as a sanity check
LGTM!

**Pull request merged**